Earthfile

```diff
@@ -22,9 +22,9 @@ ARG CGO_ENABLED=0
 # renovate: datasource=docker depName=quay.io/kairos/osbuilder-tools versioning=semver-coerced
 ARG OSBUILDER_VERSION=v0.5.3
 ARG OSBUILDER_IMAGE=quay.io/kairos/osbuilder-tools:$OSBUILDER_VERSION
-ARG GOLINT_VERSION=1.47.3
+ARG GOLINT_VERSION=1.52.2
 # renovate: datasource=docker depName=golang
-ARG GO_VERSION=1.18
+ARG GO_VERSION=1.20.2
```

Comment:
I guess it should be 1.20 as that will get updated with minor version automatically...unless we want to manually update it on minor versions in case something breaks?